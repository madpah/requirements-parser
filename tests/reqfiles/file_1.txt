file:///path/to/your/lib/project#egg=MyProject
file://../../lib/project#egg=MyProject
path/to/SomeProject#egg=SomeOtherProject
