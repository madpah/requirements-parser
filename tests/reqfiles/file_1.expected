[
 {
  "specifier": false, 
  "local_file": true, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "file:///path/to/your/lib/project", 
  "extras": [], 
  "vcs": null, 
  "path": null, 
  "line": "file:///path/to/your/lib/project#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": true, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "file://../../lib/project", 
  "extras": [], 
  "vcs": null, 
  "path": null,
  "line": "file://../../lib/project#egg=MyProject", 
  "specs": [], 
  "revision": null
 },
 {
  "specifier": false, 
  "local_file": true, 
  "name": "SomeOtherProject", 
  "editable": false, 
  "uri": null, 
  "extras": [], 
  "vcs": null, 
  "path": "path/to/SomeProject",
  "line": "path/to/SomeProject#egg=SomeOtherProject",
  "specs": [], 
  "revision": null
 }
]
